app/dashboard/templates/onepager/send2.html

```diff
@@ -26,6 +26,8 @@
   <script src="{% static "v2/js/lib/secrets.min.js" %}"></script>
   <script src="{% static "v2/js/ethereumjs-accounts.js" %}"></script>
   <script src="/dynamic/js/tokens_dynamic.js"></script>
+  <script src="{% static "v2/js/lib/bootstrap.min.js" %}"></script>
+  <script src="{% static "v2/js/bootstrap-duration-picker.min.js" %}"></script>
```

Comment:
Can we avoid adding a new lib? Also I don't see it where is used seems on the video the inputs are input type number

Reply:
It is mainly used to get the exact time units to be sent to the stream deposit. Otherwise, they need to be calculated manually (days+hours+min) -> seconds

[send.js: Line 118](https://github.com/gitcoinco/web/pull/5976/files#diff-91ab962b87bcc37ae7ecf91c849e420eR118)
```
$('#duration').durationPicker({...
```